templates/visuals/rvisual/src/visual.ts

```diff
@@ -0,0 +1,53 @@
+module powerbi.extensibility.visual {
+
+    export interface ScriptResult {
+        source: string;
+        provider: string;
+    }
+    
+    export class Visual implements IVisual {
+        private imageDiv: d3.Selection<HTMLElement>;
+        private imageElement: d3.Selection<HTMLElement>;
+
```

Comment:
do you really need d3? Reason i point this out, we will remove d3 by default in future versions, and including d3 just to do selection doesn't seem very useful since you can do it via raw DOM APIs